lib/grinder.dart

```diff
@@ -1,41 +1,6 @@
 // Copyright 2013 Google. All rights reserved. Use of this source code is
 // governed by a BSD-style license that can be found in the LICENSE file.
 
-/**
- * A library and tool to drive a command-line build.
- *
- * Grinder build files are entirely specified in Dart code. This allows you to
- * write and debug your build files with the same tools you use for the rest of
- * your project source.
- *
- * Generally, a Grinder implementation will look something like this:
- *     void main([List<String> args]) {
- *       task('init', init);
- *       task('compile', compile, ['init']);
- *       task('deploy', deploy, ['compile']);
- *       task('docs', deploy, ['init']);
- *       task('all', ['deploy', 'docs']);
- *
- *       startGrinder(args);
- *     }
- *
- *     init(GrinderContext context) {
- *       context.log("I set things up");
- *     }
- *
- *     ...
- *
- * Tasks to run are specified on the command line. If a task has dependencies,
- * those dependent tasks are run before the specified task.
- *
- * ## Command-line usage
- *     usage: dart grinder.dart <options> target1 target2 ...
- *
- *     valid options:
- *     --version     print the version of grinder
- *     -h, --help    show targets but don't build
- *     -d, --deps    display the dependencies of targets
- */
```

Comment:
Done.  I linked to the pub package page, since I didn't know how to link the README page in the dartdocs, and the package page is the most likely README to match the version of the docs they are looking at.